rust/core-lib/src/view.rs

```diff
@@ -11,25 +11,26 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+#![allow(clippy::range_plus_one)]
```

Comment:
I would prefer to just fix whatever this was pointing to?

Reply:
Yeah, there was one case where changing this would return `RangeInclusive` while `Range` is expected.

Reply:
okay, cool.

Reply:
I'll open an issue for this once this PR gets merged. I think it's a beginner-friendly refactoring issue